docroot/themes/contrib/civictheme/includes/listing.inc

```diff
@@ -75,17 +75,9 @@ function _civictheme_listing_element(Paragraph $paragraph) {
     $view->initHandlers();
     $element = [];
     // Show exposed form if available for this block display.
-    if ($show_filters) {
```

Comment:
This no longer works as this level - we need to handle this in `views_view` preprocessing